# 超音波センサ(L-MESのみのみ)SET
float64 UltraSonic0
float64 UltraSonic1
---

